kagglesdk/competitions/types/competition_enums.py

```diff
@@ -8,6 +8,7 @@ class CompetitionListTab(enum.Enum):
   COMPETITION_LIST_TAB_HOSTED = 3
   COMPETITION_LIST_TAB_UNLAUNCHED = 4
   COMPETITION_LIST_TAB_UNLAUNCHED_COMMUNITY = 5
+  COMPETITION_LIST_TAB_DEFAULT = 6
```

Comment:
This doesn't line up with the internal proto definition.  Why is this being done?  What will happen when someone adds a new enum in the internal proto that is also id=6?

Reply:
It will align once my C# PR is merged to make it so. This value is derived from the internal proto via `kapigen`.